core/templates/pages/exploration-player-page/services/extract-image-filenames-from-model.service.ts

```diff
@@ -115,9 +115,17 @@ export class ExtractImageFilenamesFromModelService {
    * @param {object} state - The state whose solution's html should be
    *                         returned.
    */
-  _getSolutionHtml(state: State): string {
+  _getSolutionHtml(state: State): string | null {
     let languageCode = (
       this.contentTranslationLanguageService.getCurrentContentLanguageCode());
+
+    // The state.interaction.solution variable threw 'Object is possibly null
+    // error'. So to prevent this error there is a check to see that if
+    // solution is null, which means that the solution in the state does not
+    // exist. If it is null then function returns null.
+    if (state.interaction.solution === null) {
+      return null;
+    }
```

Comment:
Is it valid if the solution does not exist?

Reply:
Yes, the same explanation as before.